packages/node/src/client.ts

```diff
@@ -140,9 +141,14 @@ export class NodeClient extends BaseClient<NodeClientOptions> {
    */
   protected _prepareEvent(event: Event, hint: EventHint, scope?: Scope): PromiseLike<Event | null> {
     event.platform = event.platform || 'node';
-    if (this.getOptions().serverName) {
-      event.server_name = this.getOptions().serverName;
-    }
+    event.contexts = {
+      ...event.contexts,
+      runtime: {
+        name: 'node',
+        version: global.process.version,
+      },
+    };
+    event.server_name = this.getOptions().serverName || global.process.env.SENTRY_NAME || os.hostname();
```

Comment:
What do you think about putting this event mutation into a default integration instead of in the `prepareEvent` step here.

This is the pattern most of the other SDKs follow for this.

Reply:
Interesting idea. I looked but didn't actually find any examples where we're using an integration to just set a few pieces of metadata. Which ones were you thinking of?

TBH, my gut feeling on it is that it feels kinda like overkill - it's a lot of machinery to wrap two lines of code, no? (Also, I know bundle size isn't _as_ much of a thing on the node side of things, but given serverless functions it's still something to be at least vaguely aware of, so I'd only want to do this if it seemed worth the extra code.)

Let's wait and see if anyone complains, and then if we need to pull it into an integration so that people can shut it off, we can do that.

WDYT?

Reply:
Sure that sounds fine to me!